client/keys/add.go

```diff
@@ -89,11 +89,15 @@ input
 output
 	- armor encrypted private key (saved to file)
 */
-func runAddCmd(cmd *cobra.Command, args []string) error {
+func runAddCmd(_ *cobra.Command, args []string) error {
 	var kb keys.Keybase
 	var err error
 	var encryptPassword string
 
+	if len(args) < 1 {
```

Comment:
This should be done by a `RangeArgs` or `ExactArgs` on cobra of the add cmd, no need to check for arg length here.

Reply:
I know that is checked by cobra before, but a bit of defensive programming is useful. A mistake there leads to a panic.